packages/teams-js/src/internal/videoEffectsUtils.ts

```diff
@@ -297,6 +301,58 @@ class OneTextureMetadata {
   public get audioInferenceResult(): Uint8Array | undefined {
     return this.metadataMap.get(this.AUDIO_INFERENCE_RESULT_STREAM_ID);
   }
+
+  /**
+   * @hidden
+   * Additional attributes on the video frame are string-indexed, with their stream Id dynamically generated.
+   * The mapping of attribute Ids to their stream Ids is itself stored as frame metadata with layout:
+   *
+   * | attribute count  | attribute stream Id  | attribute id                                              | ...   |
+   * | :---:            | :---:                | :---:                                                     | :---: |
+   * | 4 bytes          | 4 bytes              | variable length string (null terminated, 4 byte aligned)  | ...   |
+   *
+   *
+   * @internal
+   * Limited to Microsoft-internal use
+   */
+  public get attributes(): ReadonlyMap<string, Uint8Array> | undefined {
```

Comment:
I don't really know how I would catch a bug in this code because I don't really know enough about how your Uint8Array is supposed to be formatted. I'll do my best to read through and understand this, but I would *strongly* encourage you to have someone on your team read this and make sure that it looks correct to them.